app/src/main/java/com/github/codetanzania/ui/fragment/AppIntroBaseFragment.java

```diff
@@ -0,0 +1,147 @@
+package com.github.codetanzania.ui.fragment;
+
+import android.os.Bundle;
+import android.support.annotation.ColorInt;
+import android.support.annotation.LayoutRes;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import com.github.paolorotolo.appintro.ISlideBackgroundColorHolder;
+import com.github.paolorotolo.appintro.ISlideSelectionListener;
+import com.github.paolorotolo.appintro.util.CustomFontCache;
+import com.github.paolorotolo.appintro.util.LogHelper;
+
+// WORK AROUND
+// For some reason, the app intro library has a buggy update. This is a copy of the buggy fragment
+// with some minor difference which are introduced in order to fix the bug.
+public abstract class AppIntroBaseFragment extends Fragment implements ISlideSelectionListener,
```

Comment:
Why did we update the library? Is there a specific feature oriented reason that we need it updated?